app/models/concerns/exportable.rb

```diff
@@ -158,17 +158,11 @@ def get_field_value_for_serialization(field)
   end
 
   def external_pledge_count
-    external_pledges.count
+    external_pledges.size
   end
 
   def external_pledge_sum
-    sum = 0
-    all_pledges = external_pledges.all
-
-    all_pledges.each do |pledge|
-      sum += pledge.try :amount
-    end
-    sum
+    external_pledges.inject { |x, acc = 0| acc = x.try(:amount) || 0}
```

Comment:
`external_pledges.all` was running an additional select * from external pledges, despite eager loading. So we turn this into a snappy one line reducer.